config/schemas/config.yaml

```
@@ -4001,6 +4001,15 @@ properties:
       customReceivers:
         title: Alert Custom Receivers
         type: array
```

A bit out of scope but I saw you added a link to the routes configuration so would be nice to also have it for receivers :)
```suggestion
        description: |-
          Additional receivers that will be added to the configuration of alertmanager
          > [!note]
          > See [the upstream documentation](https://prometheus.io/docs/alerting/latest/configuration/#receiver) for reference.
        default: []
        type: array
```

---

https://github.com/elastisys/compliantkubernetes-apps/pull/2348/commits/da8562dc566587f4ffc6e204872571bb1c3f7ce8